src/shared/util/field_default_constants.h

```diff
@@ -29,20 +29,16 @@
 class FieldLine;
 class FieldCircularArc;
 
-namespace FieldConstantsRoboCup2014 {
-
-const double kFieldLength = 9000.0;
-const double kFieldWidth = 6000.0;
-const double kGoalWidth = 1000.0;
-const double kGoalDepth = 200.0;
-const double kBoundaryWidth = 250.0;
-
-extern const std::size_t kNumFieldLines;
-extern const FieldLine kFieldLines[];
-
-extern const std::size_t kNumFieldArcs;
-extern const FieldCircularArc kFieldArcs[];
-
-extern const GVector::vector2d<double> kCameraControlPoints[4][4];
+namespace FieldConstantsRoboCup2018A {
```

Comment:
is this division A parameters ?
If it's division-A, field length, field width, and goal width should be change to new numbers.

Reply:
oh yes, sry. I missed that.
I've updated it with the correct dimensions for div A. The dimension of the center circle is not clear yet, but at least I moved it to the constants too.
Anyway, this are only the defaults when initially starting the vision.